core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.html

```diff
@@ -20,7 +20,7 @@ <h1>
           </span>
         </h1>
         <li class="oppia-exploration-info-icon e2e-test-exploration-info-icon">
-          <i class="material-icons oppia-navbar-breadcrumb-icon oppia-footer-info-icon">&#xE88E;</i>
+          <i class="fas fa-info-circle oppia-navbar-breadcrumb-icon oppia-footer-info-icon"></i>
```

Comment:
The icon is part of a feature that is currently gated (not enabled by default). To enable it, you need to:
1. Log in as super admin on your local server.
2. Head to the config tab of the admin page.
3. Enabled the 3rd option (The checkpoints feature one).
![image](https://user-images.githubusercontent.com/78612244/194523445-7fae1ec0-ca17-4e32-b34d-ddb3f87e1418.png)
4. Scroll all the way down and save changes.


The icon you're looking for will then show up on the bottom right. It appears in the exploration player page.
![image](https://user-images.githubusercontent.com/78612244/194521429-329e827f-0630-4547-90d6-989fb9854076.png)

Reply:
Thanks. Updating the PR with the same.